Can't load data from minio without ssl

<!-- (At least include the following, feel free to add if you have more content) -->

### Steps to reproduce the behavior (Required)
```sql
LOAD LABEL test.test
(
    DATA INFILE("s3a://xxx")
    INTO TABLE test
)
WITH BROKER 'broker'
(
    "fs.s3a.access.key" = "xx",
    "fs.s3a.secret.key" = "xx",
    "fs.s3a.endpoint" = "xx",
    "fs.s3a.connection.ssl.enabled" = "false" # this is invalid
)
```

### Expected behavior (Required)
Load data from minio success
### Real behavior (Required)
```java
com.starrocks.broker.hdfs.BrokerException: org.apache.hadoop.fs.s3a.AWSClientIOException: doesBucketExist on dev-lake: com.amazonaws.SdkClientException: Unable to execute HTTP request: Unsupported or unrecognized SSL message: Unable to execute HTTP request: Unsupported or unrecognized SSL message
	at com.starrocks.broker.hdfs.FileSystemManager.getS3AFileSystem(FileSystemManager.java:463)
	at com.starrocks.broker.hdfs.FileSystemManager.getFileSystem(FileSystemManager.java:195)
	at com.starrocks.broker.hdfs.FileSystemManager.listPath(FileSystemManager.java:694)
	at com.starrocks.broker.hdfs.HDFSBrokerServiceImpl.listPath(HDFSBrokerServiceImpl.java:80)
	at com.starrocks.thrift.TFileBrokerService$Processor$listPath.getResult(TFileBrokerService.java:815)
	at com.starrocks.thrift.TFileBrokerService$Processor$listPath.getResult(TFileBrokerService.java:795)
	at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:38)
	at org.apache.thrift.TBaseProcessor.process(TBaseProcessor.java:38)
	at org.apache.thrift.server.TThreadPoolServer$WorkerProcess.run(TThreadPoolServer.java:248)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.apache.hadoop.fs.s3a.AWSClientIOException: doesBucketExist on dev-lake: com.amazonaws.SdkClientException: Unable to execute HTTP request: Unsupported or unrecognized SSL message: Unable to execute HTTP request: Unsupported or unrecognized SSL message
	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:144)
	at org.apache.hadoop.fs.s3a.S3AFileSystem.verifyBucketExists(S3AFileSystem.java:341)
	at org.apache.hadoop.fs.s3a.S3AFileSystem.initialize(S3AFileSystem.java:280)
	at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3247)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:472)
	at com.starrocks.broker.hdfs.FileSystemManager.getS3AFileSystem(FileSystemManager.java:457)
	... 11 more
Caused by: com.amazonaws.SdkClientException: Unable to execute HTTP request: Unsupported or unrecognized SSL message
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleRetryableException(AmazonHttpClient.java:1114)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1064)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:743)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:717)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:699)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:667)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:649)
	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:513)
	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:4325)
	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:4272)
	at com.amazonaws.services.s3.AmazonS3Client.headBucket(AmazonS3Client.java:1337)
	at com.amazonaws.services.s3.AmazonS3Client.doesBucketExist(AmazonS3Client.java:1277)
	at org.apache.hadoop.fs.s3a.S3AFileSystem.verifyBucketExists(S3AFileSystem.java:329)
	... 15 more
Caused by: javax.net.ssl.SSLException: Unsupported or unrecognized SSL message
	at sun.security.ssl.SSLSocketInputRecord.handleUnknownRecord(SSLSocketInputRecord.java:455)
	at sun.security.ssl.SSLSocketInputRecord.decode(SSLSocketInputRecord.java:184)
	at sun.security.ssl.SSLTransport.decode(SSLTransport.java:109)
	at sun.security.ssl.SSLSocketImpl.decode(SSLSocketImpl.java:1392)
	at sun.security.ssl.SSLSocketImpl.readHandshakeRecord(SSLSocketImpl.java:1300)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:435)
	at com.amazonaws.thirdparty.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:394)
	at com.amazonaws.thirdparty.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:353)
	at com.amazonaws.http.conn.ssl.SdkTLSSocketFactory.connectSocket(SdkTLSSocketFactory.java:132)
	at com.amazonaws.thirdparty.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:141)
	at com.amazonaws.thirdparty.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at sun.reflect.GeneratedMethodAccessor7.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.amazonaws.http.conn.ClientConnectionManagerFactory$Handler.invoke(ClientConnectionManagerFactory.java:76)
	at com.amazonaws.http.conn.$Proxy11.connect(Unknown Source)
	at com.amazonaws.thirdparty.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at com.amazonaws.thirdparty.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at com.amazonaws.thirdparty.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at com.amazonaws.thirdparty.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at com.amazonaws.thirdparty.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at com.amazonaws.thirdparty.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at com.amazonaws.http.apache.client.impl.SdkHttpClient.execute(SdkHttpClient.java:72)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1236)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1056)
	... 26 more
```
### StarRocks version (Required)
 - `2.1.4 d965a4f`

--- a/fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java
+++ b/fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java
@@ -105,6 +105,7 @@ public class FileSystemManager {
     private static final String FS_S3A_ENDPOINT = "fs.s3a.endpoint";
     // This property is used like 'fs.hdfs.impl.disable.cache'
     private static final String FS_S3A_IMPL_DISABLE_CACHE = "fs.s3a.impl.disable.cache";
+    private static final String FS_S3A_CONNECTION_SSL_ENABLED = "fs.s3a.connection.ssl.enabled";
 
     // arguments for ks3
     private static final String FS_KS3_ACCESS_KEY = "fs.ks3.AccessKey";
@@ -138,13 +139,13 @@ public class FileSystemManager {
     private static final String FS_OBS_IMPL = "fs.obs.impl";
 
     private ScheduledExecutorService handleManagementPool = Executors.newScheduledThreadPool(2);
-    
+
     private int readBufferSize = 128 << 10; // 128k
     private int writeBufferSize = 128 << 10; // 128k
-    
+
     private ConcurrentHashMap<FileSystemIdentity, BrokerFileSystem> cachedFileSystem;
     private ClientContextManager clientContextManager;
-    
+
     public FileSystemManager() {
         cachedFileSystem = new ConcurrentHashMap<>();
         clientContextManager = new ClientContextManager(handleManagementPool);
@@ -171,22 +172,22 @@ public class FileSystemManager {
 
         return finalPrincipal;
     }
-    
+
     /**
      * visible for test
-     * 
+     *
      * @param path
      * @param properties
      * @return BrokerFileSystem with different FileSystem based on scheme
-     * @throws URISyntaxException 
-     * @throws Exception 
+     * @throws URISyntaxException
+     * @throws Exception
      */
     public BrokerFileSystem getFileSystem(String path, Map<String, String> properties) {
         WildcardURI pathUri = new WildcardURI(path);
         String scheme = pathUri.getUri().getScheme();
         if (Strings.isNullOrEmpty(scheme)) {
             throw new BrokerException(TBrokerOperationStatusCode.INVALID_INPUT_FILE_PATH,
-                "invalid path. scheme is null");
+                    "invalid path. scheme is null");
         }
         BrokerFileSystem brokerFileSystem = null;
         if (scheme.equals(HDFS_SCHEME) || scheme.equals(VIEWFS_SCHEME)) {
@@ -203,18 +204,18 @@ public class FileSystemManager {
             brokerFileSystem = getOBSFileSystem(path, properties);
         } else {
             throw new BrokerException(TBrokerOperationStatusCode.INVALID_INPUT_FILE_PATH,
-                "invalid path. scheme is not supported");
+                    "invalid path. scheme is not supported");
         }
         return brokerFileSystem;
     }
 
     /**
      * visible for test
-     *
+     * <p>
      * file system handle is cached, the identity is host + username_password
      * it will have safety problem if only hostname is used because one user may specify username and password
      * and then access hdfs, another user may not specify username and password but could also access data
-     *
+     * <p>
      * Configs related to viewfs in core-site.xml and hdfs-site.xml should be copied to the broker conf directory.
      *
      * @param path
@@ -232,7 +233,7 @@ public class FileSystemManager {
                 logger.info("no schema and authority in path. use fs.defaultFs");
             } else {
                 logger.warn("invalid hdfs path. authority is null,path:" + path);
-                throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                         "invalid hdfs path. authority is null");
             }
         }
@@ -240,12 +241,12 @@ public class FileSystemManager {
         String password = properties.getOrDefault(PASSWORD_KEY, "");
         String dfsNameServices = properties.getOrDefault(DFS_NAMESERVICES_KEY, "");
         String authentication = properties.getOrDefault(CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION,
-            AUTHENTICATION_SIMPLE);
+                AUTHENTICATION_SIMPLE);
         if (Strings.isNullOrEmpty(authentication) || (!authentication.equals(AUTHENTICATION_SIMPLE)
-            && !authentication.equals(AUTHENTICATION_KERBEROS))) {
+                && !authentication.equals(AUTHENTICATION_KERBEROS))) {
             logger.warn("invalid authentication:" + authentication);
             throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
-                "invalid authentication:" + authentication);
+                    "invalid authentication:" + authentication);
         }
         String hdfsUgi = username + "," + password;
         FileSystemIdentity fileSystemIdentity = null;
@@ -260,11 +261,11 @@ public class FileSystemManager {
             } else if (properties.containsKey(KERBEROS_KEYTAB_CONTENT)) {
                 kerberosContent = properties.get(KERBEROS_KEYTAB_CONTENT);
             } else {
-                throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                         "keytab is required for kerberos authentication");
             }
             if (!properties.containsKey(KERBEROS_PRINCIPAL)) {
-                throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                         "principal is required for kerberos authentication");
             } else {
                 kerberosContent = kerberosContent + properties.get(KERBEROS_PRINCIPAL);
@@ -275,7 +276,7 @@ public class FileSystemManager {
                 String kerberosUgi = new String(result);
                 fileSystemIdentity = new FileSystemIdentity(host, kerberosUgi);
             } catch (NoSuchAlgorithmException e) {
-                throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                         e.getMessage());
             }
         }
@@ -299,7 +300,7 @@ public class FileSystemManager {
                 // TODO get this param from properties
                 // conf.set("dfs.replication", "2");
                 String tmpFilePath = null;
-                if (authentication.equals(AUTHENTICATION_KERBEROS)){
+                if (authentication.equals(AUTHENTICATION_KERBEROS)) {
                     conf.set(CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION,
                             AUTHENTICATION_KERBEROS);
 
@@ -322,7 +323,7 @@ public class FileSystemManager {
                         fileOutputStream.close();
                         keytab = tmpFilePath;
                     } else {
-                        throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                        throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                                 "keytab is required for kerberos authentication");
                     }
                     UserGroupInformation.setConfiguration(conf);
@@ -330,11 +331,11 @@ public class FileSystemManager {
                     if (properties.containsKey(KERBEROS_KEYTAB_CONTENT)) {
                         try {
                             File file = new File(tmpFilePath);
-                            if(!file.delete()){
-                                logger.warn("delete tmp file:" +  tmpFilePath + " failed");
+                            if (!file.delete()) {
+                                logger.warn("delete tmp file:" + tmpFilePath + " failed");
                             }
                         } catch (Exception e) {
-                            throw new  BrokerException(TBrokerOperationStatusCode.FILE_NOT_FOUND,
+                            throw new BrokerException(TBrokerOperationStatusCode.FILE_NOT_FOUND,
                                     e.getMessage());
                         }
                     }
@@ -343,15 +344,15 @@ public class FileSystemManager {
                     // ha hdfs arguments
                     final String dfsHaNameNodesKey = DFS_HA_NAMENODES_PREFIX + dfsNameServices;
                     if (!properties.containsKey(dfsHaNameNodesKey)) {
-                        throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                        throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                                 "load request missed necessary arguments for ha mode");
                     }
-                    String dfsHaNameNodes =  properties.get(dfsHaNameNodesKey);
+                    String dfsHaNameNodes = properties.get(dfsHaNameNodesKey);
                     conf.set(DFS_NAMESERVICES_KEY, dfsNameServices);
                     conf.set(dfsHaNameNodesKey, dfsHaNameNodes);
                     String[] nameNodes = dfsHaNameNodes.split(",");
                     if (nameNodes == null) {
-                        throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                        throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                                 "invalid " + dfsHaNameNodesKey + " configuration");
                     } else {
                         for (String nameNode : nameNodes) {
@@ -359,7 +360,7 @@ public class FileSystemManager {
                             String nameNodeRpcAddress =
                                     DFS_HA_NAMENODE_RPC_ADDRESS_PREFIX + dfsNameServices + "." + nameNode;
                             if (!properties.containsKey(nameNodeRpcAddress)) {
-                                throw  new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
+                                throw new BrokerException(TBrokerOperationStatusCode.INVALID_ARGUMENT,
                                         "missed " + nameNodeRpcAddress + " configuration");
                             } else {
                                 conf.set(nameNodeRpcAddress, properties.get(nameNodeRpcAddress));
@@ -381,14 +382,14 @@ public class FileSystemManager {
                     }
                     if (properties.containsKey(DFS_HA_NAMENODE_KERBEROS_PRINCIPAL_PATTERN)) {
                         conf.set(DFS_HA_NAMENODE_KERBEROS_PRINCIPAL_PATTERN,
-                            properties.get(DFS_HA_NAMENODE_KERBEROS_PRINCIPAL_PATTERN));
+                                properties.get(DFS_HA_NAMENODE_KERBEROS_PRINCIPAL_PATTERN));
                     }
                 }
 
                 conf.set(FS_HDFS_IMPL_DISABLE_CACHE, "true");
                 FileSystem dfsFileSystem = null;
                 if (authentication.equals(AUTHENTICATION_SIMPLE) &&
-                    properties.containsKey(USER_NAME_KEY) && !Strings.isNullOrEmpty(username)) {
+                        properties.containsKey(USER_NAME_KEY) && !Strings.isNullOrEmpty(username)) {
                     // Use the specified 'username' as the login name
                     UserGroupInformation ugi = UserGroupInformation.createRemoteUser(username);
                     dfsFileSystem = ugi.doAs(new PrivilegedExceptionAction<FileSystem>() {
@@ -413,8 +414,9 @@ public class FileSystemManager {
 
     /**
      * visible for test
-     *
+     * <p>
      * file system handle is cached, the identity is endpoint + bucket + accessKey_secretKey
+     *
      * @param path
      * @param properties
      * @return
@@ -427,6 +429,7 @@ public class FileSystemManager {
         String secretKey = properties.getOrDefault(FS_S3A_SECRET_KEY, "");
         String endpoint = properties.getOrDefault(FS_S3A_ENDPOINT, "");
         String disableCache = properties.getOrDefault(FS_S3A_IMPL_DISABLE_CACHE, "true");
+        String connectionSSLEnabled = properties.getOrDefault(FS_S3A_CONNECTION_SSL_ENABLED, "true");
         // endpoint is the server host, pathUri.getUri().getHost() is the bucket
         // we should use these two params as the host identity, because FileSystem will cache both.
         String host = S3A_SCHEME + "://" + endpoint + "/" + pathUri.getUri().getHost();
@@ -454,6 +457,7 @@ public class FileSystemManager {
                 conf.set(FS_S3A_SECRET_KEY, secretKey);
                 conf.set(FS_S3A_ENDPOINT, endpoint);
                 conf.set(FS_S3A_IMPL_DISABLE_CACHE, disableCache);
+                conf.set(FS_S3A_CONNECTION_SSL_ENABLED, connectionSSLEnabled);
                 FileSystem s3AFileSystem = FileSystem.get(pathUri.getUri(), conf);
                 fileSystem.setFileSystem(s3AFileSystem);
             }
@@ -465,6 +469,7 @@ public class FileSystemManager {
             fileSystem.getLock().unlock();
         }
     }
+
     /**
      * visible for test
      * <p>
@@ -584,8 +589,9 @@ public class FileSystemManager {
 
     /**
      * visible for test
-     *
+     * <p>
      * file system handle is cached, the identity is endpoint + bucket + accessKey_secretKey
+     *
      * @param path
      * @param properties
      * @return
@@ -726,12 +732,12 @@ public class FileSystemManager {
         } catch (Exception e) {
             logger.error("errors while get file status ", e);
             fileSystem.closeFileSystem();
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     e, "unknown error when get file status");
         }
         return resultFileStatus;
     }
-    
+
     public void deletePath(String path, Map<String, String> properties) {
         WildcardURI pathUri = new WildcardURI(path);
         BrokerFileSystem fileSystem = getFileSystem(path, properties);
@@ -741,16 +747,16 @@ public class FileSystemManager {
         } catch (IOException e) {
             logger.error("errors while delete path " + path);
             fileSystem.closeFileSystem();
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     e, "delete path {} error", path);
         }
     }
-    
+
     public void renamePath(String srcPath, String destPath, Map<String, String> properties) {
         WildcardURI srcPathUri = new WildcardURI(srcPath);
         WildcardURI destPathUri = new WildcardURI(destPath);
         if (!srcPathUri.getAuthority().trim().equals(destPathUri.getAuthority().trim())) {
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     "only allow rename in same file system");
         }
         BrokerFileSystem fileSystem = getFileSystem(srcPath, properties);
@@ -759,17 +765,17 @@ public class FileSystemManager {
         try {
             boolean isRenameSuccess = fileSystem.getDFSFileSystem().rename(srcfilePath, destfilePath);
             if (!isRenameSuccess) {
-                throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
-                        "failed to rename path from {} to {}", srcPath, destPath); 
+                throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
+                        "failed to rename path from {} to {}", srcPath, destPath);
             }
         } catch (IOException e) {
             logger.error("errors while rename path from " + srcPath + " to " + destPath);
             fileSystem.closeFileSystem();
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     e, "errors while rename {} to {}", srcPath, destPath);
         }
     }
-    
+
     public boolean checkPathExist(String path, Map<String, String> properties) {
         WildcardURI pathUri = new WildcardURI(path);
         BrokerFileSystem fileSystem = getFileSystem(path, properties);
@@ -780,11 +786,11 @@ public class FileSystemManager {
         } catch (IOException e) {
             logger.error("errors while check path exist: " + path);
             fileSystem.closeFileSystem();
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     e, "errors while check if path {} exist", path);
         }
     }
-    
+
     public TBrokerFD openReader(String clientId, String path, long startOffset, Map<String, String> properties) {
         WildcardURI pathUri = new WildcardURI(path);
         Path inputFilePath = new Path(pathUri.getPath());
@@ -799,11 +805,11 @@ public class FileSystemManager {
         } catch (IOException e) {
             logger.error("errors while open path", e);
             fileSystem.closeFileSystem();
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     e, "could not open file {}", path);
         }
     }
-    
+
     public ByteBuffer pread(TBrokerFD fd, long offset, long length) {
         FSDataInputStream fsDataInputStream = clientContextManager.getFsDataInputStream(fd);
         synchronized (fsDataInputStream) {
@@ -832,7 +838,7 @@ public class FileSystemManager {
             if (length > readBufferSize) {
                 buf = new byte[readBufferSize];
             } else {
-                buf = new byte[(int)length];
+                buf = new byte[(int) length];
             }
             try {
                 int readLength = readByteArrayFully(fsDataInputStream, buf);
@@ -851,12 +857,12 @@ public class FileSystemManager {
             }
         }
     }
-    
+
     public void seek(TBrokerFD fd, long offset) {
-        throw new BrokerException(TBrokerOperationStatusCode.OPERATION_NOT_SUPPORTED, 
+        throw new BrokerException(TBrokerOperationStatusCode.OPERATION_NOT_SUPPORTED,
                 "seek this method is not supported");
     }
-    
+
     public void closeReader(TBrokerFD fd) {
         FSDataInputStream fsDataInputStream = clientContextManager.getFsDataInputStream(fd);
         synchronized (fsDataInputStream) {
@@ -871,13 +877,13 @@ public class FileSystemManager {
             }
         }
     }
-    
+
     public TBrokerFD openWriter(String clientId, String path, Map<String, String> properties) {
         WildcardURI pathUri = new WildcardURI(path);
         Path inputFilePath = new Path(pathUri.getPath());
         BrokerFileSystem fileSystem = getFileSystem(path, properties);
         try {
-            FSDataOutputStream fsDataOutputStream = fileSystem.getDFSFileSystem().create(inputFilePath, 
+            FSDataOutputStream fsDataOutputStream = fileSystem.getDFSFileSystem().create(inputFilePath,
                     true, writeBufferSize);
             UUID uuid = UUID.randomUUID();
             TBrokerFD fd = parseUUIDToFD(uuid);
@@ -886,11 +892,11 @@ public class FileSystemManager {
         } catch (IOException e) {
             logger.error("errors while open path", e);
             fileSystem.closeFileSystem();
-            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR, 
+            throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
                     e, "could not open file {}", path);
         }
     }
-    
+
     public void pwrite(TBrokerFD fd, long offset, byte[] data) {
         FSDataOutputStream fsDataOutputStream = clientContextManager.getFsDataOutputStream(fd);
         synchronized (fsDataOutputStream) {
@@ -916,7 +922,7 @@ public class FileSystemManager {
             }
         }
     }
-    
+
     public void closeWriter(TBrokerFD fd) {
         FSDataOutputStream fsDataOutputStream = clientContextManager.getFsDataOutputStream(fd);
         synchronized (fsDataOutputStream) {
@@ -932,11 +938,11 @@ public class FileSystemManager {
             }
         }
     }
-    
+
     public void ping(String clientId) {
         clientContextManager.onPing(clientId);
     }
-    
+
     private static TBrokerFD parseUUIDToFD(UUID uuid) {
         return new TBrokerFD(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
     }
@@ -944,7 +950,7 @@ public class FileSystemManager {
     private int readByteArrayFully(FSDataInputStream is, byte[] dest) throws IOException {
         int readLength = 0;
         while (readLength < dest.length) {
-            int n = is.read(dest , readLength, dest.length - readLength);
+            int n = is.read(dest, readLength, dest.length - readLength);
             if (n <= 0) {
                 break;
             }
@@ -952,7 +958,7 @@ public class FileSystemManager {
         }
         return readLength;
     }
-    
+
     class FileSystemExpirationChecker implements Runnable {
         @Override
         public void run() {
@@ -975,6 +981,6 @@ public class FileSystemManager {
                 FileSystemManager.this.handleManagementPool.schedule(this, 60, TimeUnit.SECONDS);
             }
         }
-        
+
     }
 }

